tensorboard/main.py

```diff
@@ -261,6 +261,12 @@ def main(unused_argv=None):
     efi.inspect(FLAGS.logdir, event_file, FLAGS.tag)
     return 0
   else:
+
+    # If we don't use HTTP/1.1 then a new TCP socket and Python thread is
```

Comment:
Maybe clarify that HTTP/1.0 is the default?  Esp. since HTTP/2 does exist now.

Also, just checking but I assume we do in fact always sent a Content-Length header?

Reply:
Done and yes, everything sets content-length, assuming it's using `http_util.Respond`. We actually used to use HTTP/1.1 before we accidentally stopped doing it when we added Werkzeug. That helper was designed specifically to make sure important headers like that got set.